Automattic/newspack-popups
# npm bumps

### All Submissions:






### Changes proposed in this Pull Request:



Cherry-picked dependabot contributions.

### How to test the changes in this Pull Request:

1. Build the JS
2. Observe no issues

### Other information:
